test/dynamo/test_utils.py

```diff
@@ -260,6 +260,7 @@ def add(x, y):
     @mock.patch("time.time", return_value=0.001)
     @mock.patch("time.time_ns", return_value=100000)
     @dynamo_config.patch(specialize_float=False)
+    @mock.patch.dict(os.environ, {"EXPECTTEST_ACCEPT": "1"})
```

Comment:
hmm this doesn't seem correct, because if something changes, we will just always update what is within the assertExpectedInline without throwing an error.

Reply:
@angelayi hoo, I will fix it.